l2geth/les/api_backend.go

```diff
@@ -316,6 +317,10 @@ func (b *LesApiBackend) RPCGasCap() *big.Int {
 	return b.eth.config.RPCGasCap
 }
 
+func (b *LesApiBackend) RPCEVMTimeout() time.Duration {
```

Comment:
Same here

Reply:
no docs are on any of the other methods do you still want one here?